Sources/performance-worker/WorkerService.swift

```diff
@@ -87,7 +90,29 @@ final class WorkerService: Grpc_Testing_WorkerService.ServiceProtocol, Sendable
   ) async throws
     -> GRPCCore.ServerResponse.Stream<Grpc_Testing_WorkerService.Method.RunServer.Output>
   {
-    throw RPCError(code: .unimplemented, message: "This RPC has not been implemented yet.")
+    return ServerResponse.Stream { writer in
+      for try await message in request.messages {
+        switch message.argtype {
+        case let .some(.setup(serverConfig)):
+          try await self.serverSetup(serverConfig)
+        case let .some(.mark(mark)):
+          let response = try await self.makeStatsResponse(for: mark)
+          try await writer.write(response)
+        case .none:
+          ()
+        }
+        throw RPCError(code: .unavailable, message: "This RPC has not been implemented yet.")
+      }
+      try self.state.withLockedValue {
+        switch $0.role {
+        case .server(let server):
+          server.stopListening()
+        default:
+          throw RPCError(code: .unavailable, message: "There is no benchmark server.")
```

Comment:
We don't need to throw in this case, we can just ignore it. It'd be possible (but unusual) for a client to open the `runServer` RPC and then immediately close it. If that happened we'd throw this error unnecessarily.